Include/internal/pycore_interp.h

```diff
@@ -68,6 +68,8 @@ struct _is {
         uint64_t next_unique_id;
         /* The linked list of threads, newest first. */
         PyThreadState *head;
+        /* The thread currently executing in the __main__ module, if any. */
+        PyThreadState *main;
```

Comment:
The problem of changing this in 3.12 is that we are going to move the offsets of all the fields so it means that debuggers will need to modify the offsets by minor version for the first time. I would recommend to try to avoid this or move it somewhere at the end

Reply:
I'll move it to the end.

Reply:
Is this done or is there more to do?

Reply:
I just need to fix the ABI file.